src/main/java/core/basesyntax/db/Storage.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax.db;
+
+import core.basesyntax.model.Fruit;
+import java.util.HashMap;
+import java.util.Map;
+
+public class Storage {
+    private static Map<Fruit, Integer> storage = new HashMap<>();
+
+    public Storage() {
+    }
+
+    public Map<Fruit, Integer> getStorage() {
```

Comment:
```suggestion
    public static Map<Fruit, Integer> getStorage() {
```
we should be able to access storage without creating an object of the class